src/zope/app/i18n/messagecatalog.py

```diff
@@ -52,10 +52,14 @@ def queryMessage(self, id, default=None):
 
     def getPluralMessage(self, singular, plural, n):
         'See `IMessageCatalog`'
+        # Plural messages are not supported yet. Please report an issue kn
+        # github in case you want to implement this feature.
         raise NotImplementedError
 
     def queryPluralMessage(self, singular, plural, n, dft1=None, dft2=None):
         'See `IMessageCatalog`'
+        # Plural messages are not supported yet. Please report an issue kn
+        # github in case you want to implement this feature.
         raise NotImplementedError
```

Comment:
Why not make these `raise NotImplemented('plural messages are not supported yet')`?

(Actually, why not include a link to a GitHub ticket in the exception message?)

Reply:
I have consolidated the code to only use one error defined in interfaces, including the link.